<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeonamesOther</class>
 <widget class="QDialog" name="GeonamesOther">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geonames Other</string>
  </property>
  <widget class="QComboBox" name="selectByDescription">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="selectByDescriptionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>130</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select by description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="addNewPairLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>220</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add new ID/desc. pair to remember:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newDesc">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>105</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addNewPairButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="toBeSetLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>160</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ID/description to be set:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idToSet">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="descToSet">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>31</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>190</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Apply ID to selected thumbnails?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="linkToGeonames">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>220</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>(see &lt;a href=&quot;http://www.geonames.org/search.html&quot;&gt;http://www.geonames.org&lt;/a&gt;)</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>selectByDescription</tabstop>
  <tabstop>newID</tabstop>
  <tabstop>newDesc</tabstop>
  <tabstop>addNewPairButton</tabstop>
  <tabstop>idToSet</tabstop>
  <tabstop>descToSet</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
